Sources/Plasma/NucleusLib/pnNetCli/pnNcCli.cpp

```diff
@@ -54,14 +54,6 @@ You can contact Cyan Worlds, Inc. by email legal@cyan.com
 #include <mutex>
 #include <string>
 
-//#define NCCLI_DEBUGGING
-#ifdef NCCLI_DEBUGGING
-# pragma message("Compiling pnNetCli with debugging on")
-# define NCCLI_LOG  LogMsg
-#else
-# define NCCLI_LOG  LogMsg
-#endif
```

Comment:
The fact that it was logging in both cases appears to have been my fault: https://github.com/H-uru/Plasma/commit/8e05891be0e16bb5e6a79956a5f054bed4a1d596

We probably do want conditional logging...

Reply:
That explains why our OLD_ASYNC_LOG files are always so big 🤪

Reply:
Ah okay, makes sense. I think I was also confused because most logging in pnNcCli calls `LogMsg` directly without going through the macro. But looking at it again, the logic seems to be that unconditional `LogMsg` is used for errors and the conditional `NCCLI_LOG` for non-error debug messages.

Reply:
My guess is that `__VA_ARGS__` would solve whatever issues MinGW had circa-2012...

```cpp
// Do we need a separate non-intuitive NCCLI define for this?
#ifdef HS_DEBUGGING
#   define NCCLI_LOG(...) LogMsg(__VA_ARGS__)
#else
#   define NCCLI_LOG(...)
#endif
```